api/design/types/type.go

```diff
@@ -140,6 +140,9 @@ var ResourceVersionData = ResultType("application/vnd.hub.resource.version.data"
 		Format(FormatURI)
 		Example("webURL", "https://github.com/tektoncd/catalog/blob/main/task/buildah/0.1/buildah.yaml")
 	})
+	Attribute("hubRawURLPath", String, "Api url path to get the resource raw yaml", func() {
+		Example("hubRawURLPath", "/v1/tekton/task/buildah/raw")
```

Comment:
we should remove `v1`

also, can you improve the message a bit - `path of the api to get the raw of resource from hub apiserver`